static/js/services/delete-docs.js

```diff
@@ -78,8 +75,15 @@ var _ = require('underscore');
             return checkForDuplicates(docs);
           })
           .then(function() {
-            return DB().bulkDocs(docs);
+            return docs.map(function(doc) {
+              return {
+                _id: doc._id,
+                _rev: doc._rev,
+                _deleted: true,
+              };
```

Comment:
We definitely can't do this. This destroys the rest of the information which means that the document won't be replicated down to restricted users so they'll never see it as deleted.

Reply:
Well that's the whole drive of the PR :~)